src/js/background_scripts/message-listener-utils.js

```diff
@@ -121,32 +90,49 @@ https://github.com/fluid-lab/gamepad-navigator/blob/master/LICENSE
      */
     gamepad.messageListenerUtils.closeCurrentWindow = function () {
         chrome.windows.getCurrent(function (currentWindow) {
-            chrome.windows.getAll(function (windowsArray) {
-                // Remove the currently active window object from the windows array.
-                windowsArray = windowsArray.filter(function (window) {
-                    return window.id !== currentWindow.id;
-                });
+            chrome.windows.remove(currentWindow.id);
+        });
+    };
 
-                /**
-                 * Minimize all windows other than the current window before connecting
-                 * the gamepad.
-                 */
-                windowsArray.forEach(function (window) {
-                    var windowUpdate = { state: "minimized" };
-                    chrome.windows.update(window.id, windowUpdate);
+    /**
+     *
+     * Switch to the next or the previous window in the current window.
+     *
+     * @param {String} windowDirection - The direction in which the window focus should change,
+     *
+     */
+    gamepad.messageListenerUtils.switchWindow = function (windowDirection) {
+        chrome.windows.getAll(function (windowsArray) {
+            // Switch only if more than one window is present.
+            if (windowsArray.length > 1) {
+                // Find index of the currently active window.
+                var focusedWindowIndex = null;
+                windowsArray.forEach(function (window, index) {
+                    if (window.focused) {
```

Comment:
I would suggest using [`fluid.find`](https://docs.fluidproject.org/infusion/development/CoreAPI.html#fluidfindsource-func-deflt) here, that way the search will stop once you find the window with focus.